Library/Formula/popt.rb

```diff
@@ -1,7 +1,7 @@
 class Popt < Formula
   desc "Library like getopt(3) with a number of enhancements"
   homepage "http://rpm5.org"
-  url "http://rpm5.org/files/popt/popt-1.16.tar.gz"
+  url "https://fossies.org/linux/misc/popt-1.16.tar.gz"
```

Comment:
Looks like this is a 404.

Reply:
Sorry, I did not mean to upload that as is.
I will take care of that.. :-)

Reply:
Maybe just discard the changes to the popt.rb as here the original url http://rpm5.org/files/popt/popt-1.16.tar.gz seems to work just fine.
Sorry for the mess-up on my end here!

Reply:
No worries at all!